src/components/Trade/TradeInput.tsx

```diff
@@ -133,7 +133,7 @@ export const TradeInput = () => {
   const updatePreferredSwapper = useSwapperStore(state => state.updatePreferredSwapper)
   const fiatBuyAmount = useSwapperStore(selectBuyAmountFiat)
   const fiatSellAmount = useSwapperStore(selectSellAmountFiat)
-  const receiveAddress = useSwapperStore(selectReceiveAddress)
+  const receiveAddress = useReceiveAddress()
```

Comment:
This was broken following the receiveAddress changes in https://github.com/shapeshift/web/pull/4807

Reply:
that hook doesnt have an implementation for manual input yet, does it still work with this change?